File: Data Structures & Algorithms/duplicate-integer/submission-0.cpp
class Solution {
public:
    bool hasDuplicate(vector<int>& nums) {
        std::unordered_map<int, bool> temp;
        for(auto num : nums){
            if(temp.contains(num)){
                return true;
            }
            temp[num] = true;
        }
        return false;
    }
};

File: Data Structures & Algorithms/sort-colors/submission-0.py
class Solution:
    def sortColors(self, nums: List[int]) -> None:
        """
        Do not return anything, modify nums in-place instead.
        """
        store = [0,0,0]
        
        for num in nums:
            store[num] += 1
        
        index = 0
        for i, v in enumerate(store):
            for j in range(v):
                nums[index] = i
                index += 1
            
        